eras/conway/impl/src/Cardano/Ledger/Conway/TxInfo.hs

```diff
@@ -762,3 +764,26 @@ class
 
 instance ConwayEraPlutusTxInfo 'PlutusV3 ConwayEra where
   toPlutusChangedParameters _ x = PV3.ChangedParameters (PV3.dataToBuiltinData (toPlutusData x))
+
+-- | Translate a validity interval to POSIX time for Conway era, always using an open upper bound (strictUpperBound).
+transValidityInterval ::
+  Inject (ConwayContextError era) a =>
+  Tx era ->
+  EpochInfo (Either Text) ->
+  SystemStart ->
+  ValidityInterval ->
+  Either a PV1.POSIXTimeRange
+transValidityInterval _ epochInfo systemStart = \case
+  ValidityInterval SNothing SNothing -> pure PV1.always
+  ValidityInterval (SJust i) SNothing -> PV1.from <$> transSlotToPOSIXTime i
+  ValidityInterval SNothing (SJust i) -> do
+    t <- transSlotToPOSIXTime i
+    pure $ PV1.to t
```

Comment:
Looks like you forgot to change the implementation to the expected Conway behavior:
```suggestion
    pure $ PV1.Interval
            (PV1.LowerBound PV1.NegInf True)
            (PV1.strictUpperBound t)
```